logger.ts

```diff
@@ -15,3 +19,26 @@ if (process.env.NODE_ENV !== 'test') {
 if (process.env.EXPO_PUBLIC_DISABLE_LOGBOX) {
   LogBox.ignoreAllLogs(true);
 }
+
+let theLogger: Logger | undefined = undefined;
```

Comment:
Why can't you do the `createLogger()` call right here on assignment and just export `theLogger` and refer to it?